pkg/reconciler/route/table_test.go

```diff
@@ -1889,8 +1889,7 @@ func TestReconcile_ResponsiveGC(t *testing.T) {
 				WithConfigLabel("serving.knative.dev/route", "stale-lastpinned"),
 			),
 			rev("default", "config", 1, MarkRevisionReady,
-				WithRevName("config-00001"),
-				WithLastPinned(fakeCurTime.Add(-10*time.Minute))),
```

Comment:
just wondering why we dropped this.. doesn't seem related to the feature flag location change and wasn't mentioned in commit message / PR description; is it intentional?

Reply:
This is the annotation being added behind the feature flag.

For this test you'll see:
`Ctx:  setResponsiveGCFeature(context.Background(), cfgmap.Enabled)`
which is testing this enabled case